GSG-K3/urban-calendar
# Pop up/covid warning delay

### Content

- [x] Creating a popUp to inform the customer with postponing his/her reservation for 2 weeks #83 

### Testing
Write these following commands down in your terminal :musical_keyboard: 
- pull origin popUp/covidWarningDelay
- npm run dev
- go to http://localhost:3000/reservation on your browser.
- After you fill all the information and submitting the form 
-  you should receive an email by now that has your reservation time and date

### UI
![image](https://user-images.githubusercontent.com/56412800/88182106-74eac800-cc38-11ea-8205-1abb4fd7cfb2.png)
